mesheryctl/internal/cli/root/system/config.go

```diff
@@ -53,7 +53,10 @@ func getContexts(configFile string) ([]string, error) {
 
 	res, err := client.Do(req)
```

Comment:
Hey @luigidematteis please do make use of this function as this already takes care whether the server is running or not to make a request
https://github.com/meshery/meshery/blob/7206528d437a2f252a918fd10ab3e0d82e2dc903/mesheryctl/pkg/utils/auth.go#L67